addons/mrp/tests/test_order.py

```diff
@@ -860,6 +860,43 @@ def test_consumption_warning_1(self):
         warning.action_confirm()
         self.assertEqual(mo.state, 'done')
 
+    def test_consumption_warning_2(self):
+        """ Test that additional components are correctly computed in the Consumption
+        Warning wizard and that the 'Set Quantities & Validate' works as intended.
+        Test that removing a bom component is also correctly computed in the warning.
```

Comment:
:thinking: I already added in a test case for a deleted bom component into `test_consumption_action_set_qty_and_validate` and I think your fix is unrelated to it. I would say either leave it out of this test to make it easier to understand and/or add in some messages into your `assertEqual` to say what the expected behavior is.